lib/benchee/benchmark/collect/memory.ex

```diff
@@ -15,6 +15,18 @@ defmodule Benchee.Benchmark.Collect.Memory do
 
   @behaviour Benchee.Benchmark.Collect
 
+  defmacro compatible_stacktrace do
```

Comment:
What do you think about using `defmacrop` here to make sure this isn't used outside this module?

Reply:
I'll use `defmacrop` instead ! Good point 👍